external/README.md

```diff
@@ -0,0 +1,31 @@
+# OTE Algorithms
+
+This folder contains sub-projects implementing [OTE SDK](../ote_sdk) Task interfaces for different algorithms.
+Every sub-project is fully indepedent from each other, and each of them has its own dependencies and `init_venv.sh` script to initialize virtual environment.
+
+## Anomaly Classification
+ID | Name | GFlops | Size | Path
+------- | ------- | ------- | ------- | -------
+ote_anomaly_classification_padim | PADIM | 3.9 | 168.4 | anomaly/anomaly_classification/configs/padim/template.yaml
+ote_anomaly_classification_stfpm | STFPM | 5.6 | 21.1 | anomaly/anomaly_classification/configs/stfpm/template.yaml
+
+## Image Classification
+ID | Name | GFlops | Size | Path
+------- | ------- | ------- | ------- | -------
+MobileNet-V3-small | MobileNet-V3-small | 0.12 | 1.56 | deep-object-reid/configs/ote_custom_classification/mobilenet_v3_small/template.yaml
```

Comment:
Does it make sense to replace the path to the template with the link to make README easier to read?

Reply:
we will return to this question a bit later. For now, as template are located in submodules, it is hard to maintain links